apps/asap-server/src/entities/research-output.ts

```diff
@@ -109,6 +109,11 @@ export const parseGraphQLResearchOutput = (
     asapFunded: convertDecisionToBoolean(data.asapFunded),
     usedInPublication: convertDecisionToBoolean(data.usedInAPublication),
     pmsEmails: uniquePmsEmails,
+    labs:
+      data.labs?.map((lab) => ({
+        id: lab.id,
+        name: lab.flatData.name || '',
+      })) || [],
   };
```

Comment:
Teams endpoint doesn't default the lab's name to an empty string. It doesn't seem to be a problem now or when we do https://trello.com/c/5OfLSTzo/1491-show-labs-on-roms-detail-page

Do you think the two endpoints (team, rooms) should handle labs data the same way?

Reply:
totally agreed - you should have probably requested the change not approve :D